cacerts/plan.sh

```diff
@@ -4,33 +4,16 @@ pkg_description="\
 The Mozilla CA certificate store in PEM format (around 250KB uncompressed).
 "
 pkg_upstream_url="https://curl.haxx.se/docs/caextract.html"
+pkg_version="2019.28.08"
```

Comment:
Same comment here

Reply:
fixed, also went ahead and made it the latest since this PR is a few months old now